builder/bundles.go

```diff
@@ -480,7 +480,50 @@ func downloadRpms(packagerCmd, rpmList []string, baseDir string, maxRetries int)
 	return nil, downloadErr
 }
 
-func installBundleToFull(packagerCmd []string, buildVersionDir string, bundle *bundle, downloadRetries int) error {
+func extractRpm(baseDir string, rpm string) error {
+	path := baseDir + "/var/cache/yum/clear/"
+	newPath, err := filepath.Glob(filepath.Join(path, "clear-*"))
+	if err != nil {
+		return err
+	}
+	if len(newPath) > 0 {
+		newPath[0] = newPath[0] + "/packages"
+	} else {
+		return fmt.Errorf("cant find rpms packages")
+	}
+
+	rpmFull := rpm + "*.rpm"
+	out, err := filepath.Glob(filepath.Join(newPath[0], rpmFull))
+	if err != nil {
+		return err
+	}
+	if len(out) > 0 {
```

Comment:
I think it's safe to remove this condition. If len(out) == 0, then the loop won't run.